dataland-backend/src/main/kotlin/org/dataland/datalandbackend/validator/ValueCurrency.kt

```diff
@@ -0,0 +1,37 @@
+package org.dataland.datalandbackend.validator
+
+import jakarta.validation.Constraint
+import jakarta.validation.ConstraintValidator
+import jakarta.validation.ConstraintValidatorContext
+import jakarta.validation.Payload
+import org.dataland.datalandbackend.model.datapoints.standard.CurrencyDataPoint
+import kotlin.reflect.KClass
+
+/**
+ * Constraint Annotation for cross-validating fields "value" and "currency" of a data point containing a currency
+ */
+@Target(AnnotationTarget.CLASS)
+@Constraint(validatedBy = [ValueCurrencyValidator::class])
+annotation class ValueAndCurrency(
```

Comment:
Is there a reason why we are piggy-backing Currency Validation in this story?

Reply:
Ist jetzt in einem separatem branch und an die entsprechende story geheftet